101-sql-logical-server/azuredeploy.json

```diff
@@ -108,8 +118,7 @@
                       "[resourceId('Microsoft.Storage/storageAccounts', variables('storageName'))]"
                   ],
                   "properties": {
-                      "storageContainerPath": "[if(parameters('enableADS'), concat(reference(resourceId('Microsoft.Storage/storageAccounts', variables('storageName')), '2018-07-01').primaryEndpoints.blob, 'vulnerability-assessment'), '')]",
-                      "storageAccountAccessKey": "[if(parameters('enableADS'), listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('storageName')), '2018-02-01').keys[0].value, '')]",
+                      "storageContainerPath": "[if(and(parameters('enableADS'),parameters('enableVA')), concat(reference(resourceId('Microsoft.Storage/storageAccounts', variables('storageName')), '2018-07-01').primaryEndpoints.blob, 'vulnerability-assessment'), '')]",
```

Comment:
I think you can remove the conditional here since the resource itself is deployed with the same condition

Reply:
I removed it in the first commit and the deploy failed (you can see the first commit error https://dev.azure.com/azurequickstarts/azure-quickstart-templates/_build/results?buildId=9910&view=results)

Reply:
remove the apiVersion parameter from the reference call

Reply:
Ok